### ISSUE TITLE
Cannot interact with batteries in Power Grid

### ISSUE THREAD

**Issue by JotunSoldat** [CleverRaven/Cataclysm-DDA#70726]
### Describe the bug

Large Storage Batteries connected to the Solar Panels in the Solar Farm cannot be disconnected or taken down.

### Attach save file

[Frankfort-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/13851451/Frankfort-trimmed.tar.gz)

### Steps to reproduce

I connected placed two Large Storage Batteries adjacent to the Solar Panels in the Solar Farm location. I then "turned on" each of the Solar Panels. This created a working Power Grid. I then placed an Autoclave next to a Battery, used the Autoclave, and then Disconnected it from the Grid, and took it down. I then attempted to remove the Batteries, however it only lets me interact with the Grid, and I cannot take them down, or deconstruct them.

### Expected behavior

I expected to be able to disconnect and take down the batteries.

### Screenshots

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/114710101/861964cb-697d-4e36-a627-387201a10f7e)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/114710101/fa4e4c09-ca8b-47b7-9837-41671c8ca09d)

### Versions and configuration

Windows 10
Cataclysm DDA version 13b09b1 - cdda-windows-tiles-sounds-x64-2024-01-01-0221
"dda", "no_npc_food", "personal_portal_storms", "no_fungal_growth", "blazeindustries", "mindovermatter", "bombastic_perks", "MMA", "sees_player_retro", "translate_dialogue", "speedydex", "stats_through_kills", "StatsThroughSkills", "Tamable_Wildlife", "no_zombie_revive"

**Comment by Kamayana** [comment]
This is technically intended as batteries and solar panels get merged into one vehicle when attached to greatly reduce processing demand. That said... I feel like it wouldn't be impossible to have movable parts in spite of that, and given how many people have been bothered by this it should probably be addressed.

**Comment by PatrikLundell** [comment]
I can certainly see situations where you realize you want to rearrange your batteries and solar panels (in fact, I have a case of my own, but doubt I'd bother with it even if this issue was dealt with).

**Comment by JotunSoldat** [comment]
> batteries and solar panels get merged into one vehicle when attached to greatly reduce processing demand

Fair enough, but why can't I interact with it like a vehicle then to remove a battery? Frankly, I'd rather slightly slower performance around a huge number of solar panels to have a working system.

> situations where you realize you want to rearrange your batteries and solar panels

I was just wanting to recharge the batteries, not set up a permanent power station.

**Comment by PatrikLundell** [comment]
A better method to recharge batteries is to build a recharging station box (the bigger version) and drop your batteries into it. The recharging is fairly slow, but it has the advantage that it actually recharges rather than average charges out over the whole network. In fact, you'd probably want as many recharging boxes as you can, so recharging can be done in parallel (you obviously need to have enough juice in your energy storage/generation network to deal with that load).

Note that this is in no way a defense of the current implementation.
